doc/nrf/releases/release-notes-latest.rst

```diff
@@ -91,6 +91,76 @@ The following list summarizes the most important changes inherited from upstream
 
   * Removed the ``net_idx`` parameter from the Health Client model APIs because it can be derived (by the stack) from the ``app_idx`` parameter.
 
+* Drivers:
+
+  * Clock control:
+
+    * Fixed an issue in the nRF clock control driver that could lead to a fatal error during the system initialization, when calibration was started before kernel services became available.
+
+  * Display:
+
+    * Add support for temperature sensors in the ssd16xx driver.
+
+  * Entropy:
+
+    * Fixed a race condition in the entropy_nrf5 driver that could result in a miss of the wake up event (and caused the kernel.memory_protection.stack_random test to fail).
+
+  * Flash:
+
+    * Extended the flash API with the flash_get_parameters() function.
+    * Fixed an issue in the soc_flash_nrf driver that caused operations to fail on Bluetooth central that had multiple connections.
+    * Added support for 2 IO pin setup in the nrf_qspi_nor driver.
+    * Added support for sub-word lengths of read and write transfers in the nrf_qspi_nor driver.
+    * Improved handling of erase operations in the nrf_qspi_nor, spi_flash_at45, and spi_nor drivers. Now the operation is not started if it cannot be successfully completed.
```

Comment:
I added also "nRF" before "QSPI NOR" because it's a driver for a Nordic specific hardware, in contrary to the SPI NOR driver, which is a generic one.

Reply:
Done.

Reply:
> 
> 
> I added also "nRF" before "QSPI NOR" because it's a driver that uses a Nordic specific hardware, in contrary to the SPI NOR driver, which is a generic one.

OK. But that should probably also be changed for the name of the driver in Kconfig.